releases/release-1.20/release-notes/maps/pr-93078-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 93078
+releasenote:
+  text: 'The status of v1beta1 CRDs without "preserveUnknownFields:false" now shows a violation, "spec.preserveUnknownFields: Invalid value: true: must be false".'
```

Comment:
Interesting, @puerco why are some of them quoted and some not?

Reply:
Basically the golang encoder decides which ones to quote. I'll check to see if there is a way to force quotes on all values.

Reply:
Cool thanks you, it’s more an optical thing rather than a functional. 🤗